waspc/data/Generator/templates/server/src/_types/serialization.ts

```diff
@@ -0,0 +1,34 @@
+export type Payload = void | SuperJSONValue;
+
+// The part below was copied from SuperJSON and slightly modified:
+// https://github.com/blitz-js/superjson/blob/ae7dbcefe5d3ece5b04be0c6afe6b40f3a44a22a/src/types.ts
+//
+// We couldn't use SuperJSON's types directly because:
+//   1. They aren't exported publicly.
+//   2. They have a werid quirk that turns `SuperJSONValue` into `any`. 
+//      See why here: 
+//      https://github.com/blitz-js/superjson/pull/36#issuecomment-669239876
+//
+// We changed the code as little as possible to make future comparisons easier.
```

Comment:
@sodic If possible, annotate the pieces of code we changed to explain why we made certain changes (for future selfs)

Reply:
Done here: https://github.com/wasp-lang/wasp/pull/1252/commits/22c97514083e4573c258d9aa60630ac3e954b1c9